woocommerce/woocommerce-android
# Removed red dot - new feature IPP indicator in the more menu

Closes: #7841 


### Description

Removes a red dot - left over from moooore IPP project. Currently, this is shown almost only to the new users which doesn't bring much, especially when we have a banner on the main screen

### Testing instructions

* Do a clean install
* Notice tat red badge is gone from the "more" menu and from the payments icon inside of the more screen
